Source/ASViewController.mm

```diff
@@ -73,10 +74,14 @@ - (void)_initializeInstance
   if (_node == nil) {
     return;
   }
+
+  _node.viewControllerRoot = YES;
```

Comment:
This is fine for now, especially because it's thread-safe and faster than `-[ASDisplayNode(Convenience) closestViewController]`.

One edge case I can think of is that a root-of-view-controller node is reused and added to a different node hierarchy. It is rare, but we should document is somehow, and maybe assert in debug mode (e.g if `viewControllerRoot` is true, assert that its close view controller indeed points to itself).

Reply:
I added an assertion to `-[_ASDisplayView didMoveToSuperview]` to check for that case. Would that be enough?

Reply:
👍